dobi.yaml

```diff
@@ -2,17 +2,18 @@
 # alias
 # ===================================================
 
-alias=bb-buildingblock-build:
-  tasks: []
+alias=bb-gitversion-build:
```

Comment:
can we remove the bb prefix? I prefer to use short commands.

Reply:
i understand but i'm afraid that at the latest when dobi scripts from other building blocks are included, there will be duplicates in the names. therefore i want the defaults to be unique.